site/layouts/partials/version-banner.html

```diff
@@ -0,0 +1,18 @@
+{{ $color := "primary" }}
+{{ $latest_version := .Site.Params.url_latest_version }}
+{{ $path := strings.TrimPrefix (add hugo.WorkingDir "/") $.File.Filename -}}
+{{ $current_version := .Site.Params.version }}
+{{ if not (strings.Contains $path "docs") }}
+{{ if not (strings.Contains $path "latest") }}
+{{ if not (strings.Contains $path $current_version) }}
+<div class="alert alert-warning">
+    This version documentation for Envoy Gateway is no longer actively maintained.
+    <br/>
+    {{ with $latest_version }}
+    For up-to-date documentation, see the
+    <a href="{{ $latest_version | safeURL }}" target="_blank">{{$current_version}}</a>.</p>
```

Comment:
should be use `/docs` prefix path here instead, so they know to always go to `/docs` instead of figuring out current version ?

Reply:
it's
```
{{ $latest_version := .Site.Params.url_latest_version }}
url_latest_version = "/docs"
```

Reply:
<img width="1008" alt="image" src="https://github.com/user-attachments/assets/4282339e-41f2-42f6-8127-23474d528ea0" />